.github/workflows/pr-e2e.yml

```diff
@@ -141,6 +141,7 @@ jobs:
           AZURE_DEVOPS_PAT: ${{ secrets.AZURE_DEVOPS_PAT }}
           AZURE_DEVOPS_POOL_NAME: ${{ secrets.AZURE_DEVOPS_POOL_NAME }}
           AZURE_DEVOPS_PROJECT: ${{ secrets.AZURE_DEVOPS_PROJECT }}
+          AZURE_EVENT_HUB_CONNECTION_STRING: ${{ secrets.AZURE_EVENTHBUS_MANAGEMENT_CONNECTION_STRING }}
```

Comment:
Let's name the env as the secret, for the maintenance will be easier. I'll rename it in my PR

Reply:
```suggestion
          AZURE_EVENTHBUS_MANAGEMENT_CONNECTION_STRING : ${{ secrets.AZURE_EVENTHBUS_MANAGEMENT_CONNECTION_STRING }}
```